raitools/raitools/_managers/explainer_manager.py

```diff
@@ -126,13 +158,55 @@ def name(self):
         :return: The name of the explainer manager.
         :rtype: str
         """
-        return "explain"
+        return ManagerNames.EXPLAINER
 
     def save(self, path):
-        raise NotImplementedError(
-            "Save not implemented for ExplainerManager")
+        """Save the ExplainerManager to the given path.
+
+        :param path: The directory path to save the ExplainerManager to.
+        :type path: str
+        """
+        top_dir = Path(path)
+        # save the explanation
+        if self._explanation:
+            save_explanation(self._explanation, top_dir / EXPLANATION)
+        meta = {IS_RUN: self._is_run,
+                IS_ADDED: self._is_added}
+        with open(Path(path) / META_JSON, 'w') as file:
+            json.dump(meta, file)
 
     @staticmethod
-    def load(path):
-        raise NotImplementedError(
-            "Load not implemented for ExplainerManager")
+    def load(path, raianalyzer):
```

Comment:
Any particular reason why it's `raianalyzer` and not `rai_analyzer` ?

Reply:
done, renamed raianalyzer to rai_analyzer everywhere

Reply:
wait, why don't we call raiwidgets rai_widgets by the same logic then?

Reply:
discussed, the module name is raiwidgets, file and variables rai_widgets, to follow python conventions